justfile

```diff
@@ -49,20 +49,18 @@ py-build:
 py-format:
     black --config rerun_py/pyproject.toml {{py_folders}}
     blackdoc {{py_folders}}
-    isort {{py_folders}}
     pyupgrade --py37-plus `find rerun_py/rerun/ -name "*.py" -type f`
+    ruff --fix --config rerun_py/pyproject.toml  {{py_folders}}
 
 # Check that all the requirements.txt files for all the examples are correct
 py-requirements:
     find examples/python/ -name main.py | xargs -I _ sh -c 'cd $(dirname _) && echo $(pwd) && pip-missing-reqs . || exit 255'
 
 # Run linting
-py-lint: py-requirements
+py-lint:
```

Comment:
The dep on py-requirements still made this quite slow, so I pulled it out and added it to CI.

Reply:
Good call!